src/ui/events.js

```diff
@@ -10,29 +10,28 @@ import type Map from './map.js';
 import type LngLat from '../geo/lng_lat.js';
 
 /**
- * `MapMouseEvent` is the event type for mouse-related map events.
+ * `MapMouseEvent` is the type for objects emitted by mouse-related [`Map` events](https://docs.mapbox.com/mapbox-gl-js/api/map/#map-events).
  * @extends {Object}
  * @example
- * // The `click` event is an example of a `MapMouseEvent`.
- * // Set up an event listener on the map.
- * map.on('click', function(e) {
- *   // The event object (e) contains information like the
- *   // coordinates of the point on the map that was clicked.
- *   console.log('A click event has occurred at ' + e.lngLat);
- * });
+ * // Example of a MapMouseEvent of type "click"
+ * {
+ *     lngLat: {
+ *         lng: 40.203,
+ *         lat: -74.451
+ *     },
+ *     originalEvent: {...},
```

Comment:
I used ellipses wherever an object was really large or just a browser event being passed through. I don't _love_ this approach, but thought something like

```
{
  originalEvent: MouseEvent: {...}
}
```

or similar would be even more confusing.

Reply:
I agree that the ellipses pattern can be problematic, but in this instance where "the object-ness of the thing" and "the shape of the object" are what we're trying to convey, I think it works – and I even think that conveying "the object-ness of the thing within the thing" can be helpful. I appreciate this choice as the best one to represent a condensed, grokkable summary of the thing!

Reply:
@mapbox/gl-js team, do you have an opinion on how we can present these object most clearly?

Reply:
What you have here is pretty standard for abridging an object in an example

Reply:
Great, thank you for affirming this @ryanhamley. 

We will codify this as a standard in our documentation code style guide: https://github.com/mapbox/documentation/pull/691